ncm-openldap/src/main/pan/components/openldap/schema.pan

```diff
@@ -178,7 +178,7 @@ type ldap_global = {
 
 
 type ldap_database_string = string with match(SELF,
-    "^(bdb|config|dnssrv|hdb|ldap|ldif|meta|monitor|null|passwd|perl|relay|shell|sql)$"
+    "^(bdb|mdb|config|dnssrv|hdb|ldap|ldif|meta|monitor|null|passwd|perl|relay|shell|sql)$"
```

Comment:
The list is currently sorted alphabetically so can you maintain than please?

Reply:
fixed